scripts/login.js

```diff
@@ -0,0 +1,8 @@
+const content = document.querySelector(".input input");
```

Comment:
you can give different class-name to `.input`

Reply:
made the required changes